lib/web_ui/lib/src/engine/text/canvas_paragraph.dart

```diff
@@ -175,7 +173,7 @@ class CanvasParagraph implements ui.Paragraph {
         final RangeBox box = boxes[j++];
 
         if (box is SpanBox) {
-          lastSpanElement = html.document.createElement('span') as html.HtmlElement;
+          lastSpanElement = html.document.createElement('flt-span') as html.HtmlElement;
```

Comment:
Custom elements behave like `div`s right? Do you need `display: inline` (or `inline-block`) to retain what was special of using a `span` tag earlier? Or are we just doing display:block on everything?

Reply:
That's a good point.

The display property doesn't matter anymore. Each `<flt-span>` contains a single fragment of text that can't be broken into multiple lines.

A single line of text that is positioned absolutely shouldn't be affected by `display:block` or `display:inline`.